src/endpoints/auth/forgot-email.route.ts

```diff
@@ -36,8 +43,13 @@ export const forgotEmailRoute: RouteCustomOptions<{ Body: IForgotEmailBodySchema
     }
 
     await VerificationCodeEntity.create({ userId: String(user._id), code, expiresAt }).save();
-    await EmailService.sendMessageToEmail(email, code, EmailEnum.RECOVERY_LETTER);
 
-    return statusOutputSuccess;
+    try {
+      await EmailService.sendMessageToEmail(email, code, EmailEnum.RECOVERY_LETTER);
+    } catch (err) {
```

Comment:
Your `catch` doesn't really do anything. At least add some useful logging message (for developers) here.